docs/conversion/sakai_20_oracle_conversion.sql

```diff
@@ -84,5 +84,8 @@ ALTER TABLE SAM_ASSESSACCESSCONTROL_T ADD FEEDBACKSCORETHRESHOLD FLOAT(126);
 ALTER TABLE SAM_PUBLISHEDACCESSCONTROL_T ADD FEEDBACKSCORETHRESHOLD FLOAT(126);
 -- END SAK-42400
 
--- SAK-42498
+-- BEGIN SAK-42498
+ALTER TABLE BULLHORN_ALERTS DROP INDEX IDX_BULLHORN_ALERTS_ALERT_TYPE_TO_USER;
```

Comment:
This syntax isn't valid for Oracle [1]. Syntax is `DROP INDEX <indexName>`.

Also, the name being referenced here (`IDX_BULLHORN_ALERTS_ALERT_TYPE_TO_USER`) is too long in the Oracle world. For example, if I run `DROP INDEX IDX_BULLHORN_ALERTS_ALERT_TYPE_TO_USER;`, Oracle returns the following error:

```
Error starting at line : 1 in command -
DROP INDEX IDX_BULLHORN_ALERTS_ALERT_TYPE_TO_USER
Error report -
ORA-00972: identifier is too long
00972. 00000 -  "identifier is too long"
*Cause:    An identifier with more than 30 characters was specified.
*Action:   Specify at most 30 characters.
```

It's not possible for this index to be created with this name, but I'm not sure what it would actually be called as we don't have this index in our Oracle DBs. The only index we have on this table is a `UNIQUE` index on the `ID` column, and it has a generated name as opposed to a given name (`SYS_C0017954`).

[1] https://docs.oracle.com/cd/B28359_01/server.111/b28286/statements_8016.htm#SQLRF01510

Reply:
okay will delete this line